src/Core/Components/Dialog/FluentDialog.razor.cs

```diff
@@ -145,9 +150,9 @@ private async Task<DialogEventArgs> RaiseOnStateChangeAsync(DialogEventArgs args
     /// Displays the dialog.
     /// </summary>
     [ExcludeFromCodeCoverage]
-    public async Task ShowAsync()
+    public async Task<IJSObjectReference> ShowAsync()
```

Comment:
I don't think returning an IJSObjectReference is a good idea. This reference could be misused for other purposes.

Would it be possible to store this reference in a private variable in FluentDialog and then use it in `FocusPreviousElementAsync`?
```csharp
_previousElement = await JSRuntime.InvokeAsync<IJSObjectReference> ...
```